cvat/apps/engine/media_extractors.py

```diff
@@ -326,17 +365,21 @@ class ZipCompressedChunkWriter(IChunkWriter):
     def save_as_chunk(self, images, chunk_path):
         image_sizes = []
         with zipfile.ZipFile(chunk_path, 'x') as zip_chunk:
-            for idx, (image, _ , _) in enumerate(images):
-                w, h, image_buf = self._compress_image(image, self._image_quality)
+            for idx, (image, _, _) in enumerate(images):
+                if self._dimension == "2d":
+                    [w, h, image_buf], extension = self._compress_image(image, self._image_quality), "jpeg"
+                else:
+                    properties = get_pcd_properties(image)
+                    w, h, image_buf, extension  = int(properties["WIDTH"]), int(properties["HEIGHT"]), image, "pcd"
```

Comment:
Let's split the assignments into few lines.

Reply:
Made changes accordingly.